client/blocks/eligibility-warnings/index.tsx

```diff
@@ -235,7 +237,8 @@ export const EligibilityWarnings = ( {
 						disabled={
 							isProceedButtonDisabled( isEligible, listHolds ) ||
 							siteIsSavingSettings ||
-							siteIsLaunching
+							siteIsLaunching ||
+							isContinueButtonDisabled
 						}
 						busy={ siteIsLaunching || siteIsSavingSettings }
```

Comment:
Thought: might it also be worth showing a `busy` state for the Continue button while we're transferring? (Not a blocker.)